docs/commands/walker.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	homedir "github.com/mitchellh/go-homedir"
+)
+
+func getUserName() string {
```

Comment:
Thanks. Instead of parsing the username from the home path, there might be a way to get the username from  the system directly?